frontend/src/redux/reducers/ui.tsx

```diff
@@ -188,6 +197,14 @@ function reducer(state = _.cloneDeep(INITIAL_STATE), action: Action) {
       newState.sidebar = { ...newState.sidebar, ...setInitialSidebarOpen() };
       return newState;
     }
+    case UI_APP_LOGO_BRANDING: {
+      newFilters.branding.logo = action.appLogoComponent;
```

Comment:
This is not a component, it's a function that returns a component.

Reply:
I made a suggestion elsewhere that we pass in a component instead. This would need to be kept if we use that suggestion.